src/com/egs/arsen/task2/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.egs.arsen.task2;
+
+import com.egs.arsen.task2.core.Program;
+
+public class Main
+{
+	public static void main(String[] args)
+	{
```

Comment:
reformat code. remove C# style coding

Reply:
:D every time the same mistake , i think its not my bad , i check out the code with cntrl+L . Probably the problem is in that ?